.github/workflows/test.yaml

```diff
@@ -99,16 +161,110 @@ jobs:
         with:
           version: 9
 
-      - name: Change directory to examples/cloudflare-workers
-        run: cd examples/nextjs
-
       - name: Install Dependencies
         run: pnpm install
         working-directory: examples/nextjs
 
       - name: Install local package
-        run: pnpm install ../../dist
+        run: pnpm install @upstash/qstash@../../dist
         working-directory: examples/nextjs
 
       - name: Local build
         run: pnpm run build
+        working-directory: examples/nextjs
+
+      - name: Run Example
+        run: pnpm dev &
+        working-directory: examples/nextjs
+
+      - name: Test
+        run: bun test ci.test.ts
+        working-directory: examples/nextjs
+        env:
+          DEPLOYMENT_URL: http://localhost:3000
+
+  nextjs-deployed:
+    concurrency: nextjs-deployed
+    runs-on: ubuntu-latest
+    needs:
+      - release
+    steps:
+      - name: Setup repo
+        uses: actions/checkout@v3
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+
+      - name: Setup Bun
+        uses: oven-sh/setup-bun@v1
+        with:
+          bun-version: latest
+
+      - uses: pnpm/action-setup@v2
+        with:
+          version: latest
+
+      - name: Deploy
+        run: |
+          pnpm add @upstash/qstash@${{needs.release.outputs.version}}
+          DEPLOYMENT_URL=$(npx vercel --token=${{ secrets.VERCEL_TOKEN }})
+          echo "DEPLOYMENT_URL=${DEPLOYMENT_URL}" >> $GITHUB_ENV
+        env:
+          VERCEL_ORG_ID: ${{secrets.VERCEL_TEAM_ID}}
+          VERCEL_PROJECT_ID: "prj_dUPhtwNlX6otLH5CSV4N0CsZzoaq"
```

Comment:
can we move `VERCEL_PROJECT_ID` to action secrets? We don't need to expose it here, also would be easier to change later on.

Reply:
I think we can, but then we should do it in all CI PRs, and open one for Redis too

Shall we do it another time? :D

Reply:
Yeah I thought of that too :D 
Maybe when we need to change the id we can fix it :)